util/timeutil/time.go

```diff
@@ -0,0 +1,160 @@
+// Copyright 2017 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package timeutil
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+	"syscall"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// init initializes `locCache`.
+func init() {
+	// We need set LocalStr when it is in testing process.
+	if LocalStr == "" {
+		LocalStr = "System"
+	}
+	LocCache = &locCache{}
+	LocCache.locMap = make(map[string]*time.Location)
+}
+
+// LocCache is a simple cache policy to improve the performance of 'time.LoadLocation'.
+var LocCache *locCache
+
+// LoadSystemTZ loads system timezone from mysql.tidb and returns the value.
+var LoadSystemTZ func() string
```

Comment:
Where will this function be called outside this package ?

Reply:
We don't need to expose this.